tests/integration_test.go

```diff
@@ -27,7 +27,13 @@ func (vm *VMTest) RunTests(packages ...string) func(t *testing.T) {
 		vmtest.RunGoTestsInVM(t, packages,
 			vmtest.WithVMOpt(
 				vmtest.WithSharedDir("ovmf/keys"),
-				vmtest.WithInitramfsFiles("sbctl:bin/sbctl"),
+				vmtest.WithInitramfsFiles(
+					"sbctl:bin/sbctl",
+					"/usr/bin/openssl:bin/openssl",
+					"/usr/bin/find:bin/find",
+					"/usr/bin/head:bin/head",
+					"/usr/bin/sig-list-to-certs:bin/sig-list-to-certs",
+				),
```

Comment:
You mean, implement them in Go, build a single binary and inject that helper binary in the initramfs here (instead of the individual ones)?

That would make tests less dependent on the host OS, which is good. I just want to make sure I understand your suggestion correctly.

Reply:
Ah no, I mean it's simpler to just implement these things as helper functions instead of injecting binaries into the `initramfs`.

The helper functions can also be shared between tests which is going to be nicer as well.

Reply:
Hm, that would need me to somehow get the resulting files out of the VM to inspect them in "go space". I don't see a way in the `vmtest` library to either copy files from the VM to the host or share a directory somehow. Maybe this? https://github.com/hugelgupf/vmtest/blob/5d9f3d34a58dc7b13bca786e8ac32d3e2ce4e95d/examples/shareddir/vm_test.go#L30

Reply:
Ah, then you are misunderstanding how the tests are being run. They are run inside the VM :)

We are using `vmtest/gotest` which is using a special init process to run the built go binary test inside the VM and output a JSON format that the library is passing forward as a test.

This is why you are including `guest.SkipIfNotInVM(t)` with the test, to check if we are being run inside the qemu VM.

It's quite clever and I recommend reading over the code in `vmtest`.

https://github.com/hugelgupf/vmtest/blob/main/govmtest/gotest.go

Reply:
As an example, this if how `go-uefi` is capable of doing fairly simple tests.

https://github.com/Foxboron/go-uefi/blob/master/tests/tests/secure_boot_enabled/secure_boot_enabled_test.go

Reply:
Oh true :facepalm: ! I just can't get used to it yet. My mind keep gravitating to the types of tests I'm used to :D. 
It makes sense now. Let me change the tests.

Reply:
I tried something. I pushed it as a separate commit for easier review. We can squash later. Let me know what you think.

Reply:
Thanks! Looks much better now :)